README.md

```diff
@@ -144,6 +144,15 @@ clients can make of use of the general functionality contained in
 
 ### Development
 
+enrich-classpath is important for development of Java-related features in Orchard.
```

Comment:
Perhaps you should elaborate on how is it important exactly, so that clearer to new contributors.